README.md

```diff
@@ -2,105 +2,40 @@
 
 ## Description
 
-This repository hosts the Kubernetes Python Operator for Ory Hydra - a scalable, security first OAuth 2.0 and OpenID Connect server.
-
-For more details and documentation, visit https://www.ory.sh/docs/hydra/
+Python Operator for Ory Hydra - a scalable, security first OAuth 2.0 and OpenID Connect server. For more details and documentation, visit https://www.ory.sh/docs/hydra/
 
 ## Usage
 
-The Hydra Operator may be deployed using the Juju command line as follows:
-
-Deploy the `postgresql-k8s` charm:
-
 ```bash
 juju deploy postgresql-k8s --channel edge --trust
+juju deploy hydra --trust
```

Comment:
there's no need to deploy with `--trust`

Reply:
We're patching the k8s service aren't we? This requires k8s credentials, hence the `--trust`

Reply:
True, it's also [suggested](https://github.com/canonical/hydra-operator/blob/main/lib/charms/observability_libs/v0/kubernetes_service_patch.py#L235) in the service patch lib, but the service gets patched even without trusting the charm. I asked about it in [observability](https://chat.charmhub.io/charmhub/pl/jbdijf8z97rqpcihjkimfxbmzw).

Reply:
Yeah it depends on your k8s cluster ultimately. If you deploy your charm on GKE or other cloud k8s, you will end up having those errors:
```
unit-grafana-k8s-0: 08:29:58 ERROR unit.grafana-k8s/0.juju-log Unable to patch the Kubernetes service: Failed to patch k8s service: (403)
Reason: Forbidden
```

Reply:
That's a good point, I'll add `trust` to integration tests and kratos as well. Thanks